src/System.Net.Ping/src/System/Net/NetworkInformation/Ping.cs

```diff
@@ -272,12 +272,16 @@ public Task<PingReply> SendPingAsync(IPAddress address, int timeout, byte[] buff
                 throw new ArgumentException(SR.net_invalid_ip_addr, nameof(address));
             }
 
+#pragma warning disable CS0618 // IPAddress.Address is obsoleted, but it's the most efficient way to get the Int32 IPv4 address
```

Comment:
Yup. I thought it would not be readable by inserting inside ? : statement... doing it.